helmfile/charts/networkpolicy-service/templates/certmanager-np.yaml

```
@@ -0,0 +1,129 @@
+{{ if .Values.certManager.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: cert-manager-deny-all
+  namespace: cert-manager
+spec:
+  podSelector: {}
+  policyTypes:
+    - Ingress
+    - Egress
+
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: cert-manager-np
+  namespace: cert-manager
+spec:
+  podSelector: 
+    matchLabels:
+      app: cert-manager  
+  policyTypes:
+    - Ingress
+    - Egress
+  ingress:        
```

NIT: I see this on multiple lines, there's trailing spaces. Could you remove all those?

```suggestion
  ingress:
```

---

yes, my pre-commit isn't working.

---

You could probably set up your IDE to do this instead if you want